trieste/models/gpflow/sampler.py

```diff
@@ -839,12 +835,12 @@ def __init__(
             kernel_K(self._inducing_points, x)
         )
 
-    def call(self, x: TensorType) -> TensorType:  # [N, D] -> [N, F + M] or [L, N, F + M]
+    def call(self, inputs: TensorType) -> TensorType:  # [N, D] -> [N, F + M] or [L, N, F + M]
```

Comment:
What does the linter complain about here?

Reply:
This was overriding an ABC method (in gpflux) where the parameters are named differently.